CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+0.13.3 - 2022-12-19
+-------------------
+
+* Formally require flake8 5.0.0 or later using ``install_requires``.
```

Comment:
I don't think we need to specify `install_requires`. It's enough to say we increased the lowest supported version of flake8